src/packagedcode/citation.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import io
+import saneyaml
+from packagedcode import models
+
+"""
+Handle CITATION.cff files, see https://citation-file-format.github.io/
+https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+See https://github.com/citation-file-format/citation-file-format/tree/main/examples/1.2.0/pass for examples.
+"""
+
+class CitationHandler(models.DatafileHandler):
+    datasource_id = 'citation_cff'
+    default_package_type = 'citation'

```

Comment:
is this the best type? what about cff? and is this really a new PURL type?
There are likely details in the URLs that would make this another PURL type.